base/redactor_test.go

```diff
@@ -98,6 +98,10 @@ func TestRedactionLevelUnmarshalText(t *testing.T) {
 	assert.NoError(t, err)
 	goassert.Equals(t, level, RedactFull)
 
+	err = level.UnmarshalText([]byte("unset"))
+	assert.NoError(t, err)
+	goassert.Equals(t, level, RedactionLevelDefault)
```

Comment:
This should be `RedactUnset` really, in the same way that MarshalText returns "unset".

Want to keep the "default" handling fairly isolated and keep the JSON/Text marshaling accurate to what is being passed in/out.